### PR TITLE
Landing page tests

### PR DESCRIPTION
**Pull request by robertsapunarich**
I disabled fixtures in the test_helper file, added the factory_girl gem, and included test helpers for devise, so the `should get index` test passes and doesn't return an error message.

### CODE REVIEW

File: firehose-defcon [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit dcb7732edd6f1e67c5ba928f675ee03111574b66
```
Comment (kenmazaika) [inline firehose-defcon]: This file should be deleted, I believe.

**Comment by kenmazaika** [comment]
Made 1 comment, but besides that looks good to me.

I didn't realize we already had test code in https://github.com/FirehoseCommunity/DEFCON/blob/master/test/controllers/static_pages_controller_test.rb

Make the one small change (I believe this command will do it):

```
rm firehose-defcon
```

And commit the change and we should be ready to merge this one.

**Comment by robertsapunarich** [comment]
Sounds good. I forgot about the subproject folder. Not sure how it got there. I'll remove that, and factorygirl too.

**Comment by kenmazaika** [comment]
Feel free to leave factory girl in, it won't hurt anything and we'll need it eventually.

**Comment by kenmazaika** [comment]
We'll just need to add the specific factories later down the line.

**Comment by robertsapunarich** [comment]
Removed the folder. Should be good to go.

**Comment by kenmazaika** [comment]
The code seems good to go, but there seem to be merge conflicts.  @robertsapunarich can you resolve those conflicts and then I'll push the button to merge.

**Comment by robertsapunarich** [comment]
@kenmazaika looks like the Gemfile was missing simple_form. Consider it resolved!

**Comment by kenmazaika** [comment]
Boom.  Thumbs up, Rob!  :+1:

**Pull request merged**